src/Interactive/EditorFeatures/Core/Extensibility/Interactive/InteractiveEvaluator.cs

```diff
@@ -192,8 +192,8 @@ private void ProcessStarting(InteractiveHostOptions options)
             var metadataService = _workspace.CurrentSolution.Services.MetadataService;
             ImmutableArray<string> referencePaths;
 
-            // reset configuration:
-            if (File.Exists(_responseFilePath))
+            // reset configuration (null initialization file indicates "noconfig"):
+            if (options.InitializationFile != null && File.Exists(_responseFilePath))
```

Comment:
What's the relationship between ```options.InitializationFile``` and ```_responseFilePath```?  It seems odd to check ```options.InitializationFile```, but then not use it.